lib/RGBot.js

```diff
@@ -248,43 +515,75 @@ const RGBot = class {
         try {
             return this.mcData.items[itemId];
         } catch (err) {
-            console.error(`Couldn't find item in notch data: ${err.message}`);
+            this.#logWarn(`Couldn't find item in notch data: ${err.message}`);
             return null;
         }
     }
 
     /**
-     * Determines whether an Entity's name and/or displayName are equal to a targetName string.
+     * Determines whether an Entity's username or name is equal to a targetName string.  Does not consider displayName.
```

Comment:
I think you'd realistically pick one or the other. Originally this was nice to have because if you knew the name of an item from hovering over it in-game or looking at a wiki (this would be the display name) then you wouldn't have to go look up the name from some other resource... there are a few cases where the name and display name contain the same words in different orders so they are not always 1-to-1. 

The more we've added to this library, the more I think we should pick one, even if it's just the display name. Because some of our methods accept both while others are specifically looking for some "entity type" ie. its name